interview-questions.md

```diff
@@ -7,62 +7,63 @@ Then, research the question to expand on your answer. Even if you feel you have
 
 INSTRUCTOR EXAMPLE: What is a conditional statement?
 
-  Your answer:
+  Your answer: 
 
   Researched answer:
 
 
 
 1. What is git? What is the difference between git and Github?
 
-  Your answer:
+  Your answer: software or tool that helps developers to manage their code. Github is the online place where the code is stored.
 
-  Researched answer:
+  Researched answer: Git is a version control system and is local. It stores and tracks your code history. Github is a cloud-based platform,owned by Microsoft, that allows multiple developers to collaborate on code and see edits in real time. It can also be used as a networking site as you can set your repos to be public. It's like a portfolio of your code work that you can share with anyone, especially employers.
 
 
 
 2. Which JavaScript operators will return a Boolean value?
 
-  Your answer:
+  Your answer: conditional statements returning true or false. Uses equal signs 
+  and <,>
 
-  Researched answer:
+  Researched answer: logical, relational, and equality operators
 
 
 
 3. What is an index? What is the difference between index and value?
 
-  Your answer:
+  Your answer: Index gives the location of a character starting at 0. Values are the info inside the function
 
-  Researched answer:
+  Researched answer: Index shows the position of the element within a string. Values are used with variables. Values can be boolean, string, numbers, or arrays. If the fridge is the variable, the food inside are the values...thought of that myself...hope it makes sense, haha.
```

Comment:
Love it!  In the context of an array, the value is the data at a specific index.  So if the array was the fridge, the food is the value, and where it is placed is its index.